Detectors/Upgrades/ALICE3/Passive/include/A3DetectorsPassive/A3Pipe.h

```diff
@@ -0,0 +1,67 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICE3_PASSIVE_PIPE_H
+#define ALICE3_PASSIVE_PIPE_H
+
+#include "Rtypes.h"
+#include "A3DetectorsPassive/A3PassiveBase.h"
+
+namespace o2
+{
+namespace passive
+{
+class A3Pipe : public A3PassiveBase
```

Comment:
Personally, I am not too fond of using prefixes (like A3 here) to group classes. @sawenzel is this common practice in O2? Wouldn't it be cleaner to use a namespace for this purpose?

Reply:
Just to clarify: this is not grouping classes, it is defining a new class name for the custom beampipe, can be whatever else.

Reply:
Some extra qualification is definitely needed. I don't mind having it as part of the classname (allows for easy distinction even when we say "using namespace").

Reply:
@mconcas Maybe I was not clear, for me ALICE3::Pipe would be cleaner, also for the future. But if @sawenzel says that they actually prefer having prefixes in the class name, I can live with that. Then we should just make sure we use the same prefix everywhere, and at that point I prefer something that you can meaningful grep for, not sure how many unwanted of occurrences of A3 one gets, but your call then ..

Reply:
I'll move to the extended name, good point about "grepping"

Reply:
I have no strict preference. A namespace is good (but it needs to alice3 - namespaces are enforced to be lower case).